fluent-package/apt/commonvar.sh

```diff
@@ -16,6 +16,14 @@ case ${code_name} in
     distribution=ubuntu
     channel=universe
     mirror=http://archive.ubuntu.com/ubuntu/
+    case ${code_name} in
+        focal|jammy)
+            if [ "$architecture" = "arm64" ]; then
+                echo "For ${code_name} (arm64), use ubuntu-ports"
+                mirror=http://ports.ubuntu.com/ubuntu-ports
+            fi
+            ;;
+    esac
```

Comment:
Could you tell me why we need this?

Reply:
For example, arm64 package is not available from archive.ubuntu.com.

https://github.com/fluent/fluent-package-builder/actions/runs/13127385504/job/36626186947
https://github.com/fluent/fluent-package-builder/actions/runs/13127385504/job/36626187253

Reply:
but it might be better not to switch code_name here.

Reply:
Fixed.
(temporary piuparts execution was disable for noble, but it is same for noble)

Reply:
I see! Thanks!